docs/source/api/pystiche_papers.ulyanov_et_al_2016.rst

```diff
@@ -1,6 +1,103 @@
 ``pystiche_papers.ulyanov_et_al_2016``
 =========================================
 
+Unfortunately, the parameters in the reference implementation differ from the parameters
+described in the paper. If ``impl_params is True``, the parameters from the reference
+implementation are used instead of the parameters from the paper. The following parts
+are affected:
+
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.training`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.stylization`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.optimizer`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.lr_scheduler`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.content_transform`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.style_transform`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.batch_sampler`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.content_loss`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.style_loss`,
+  - :func:`~pystiche_papers.ulyanov_et_al_2016.transformer`.
+
+
+.. _table-branches-ulyanov_et_al_2016:
+
+The original authors have published two very similar papers, one with
+:class:`~torch.nn.BatchNorm2d` and a later one with :class:`~torch.nn.InstanceNorm2d`:
+
+  - ``master``: Corresponds to the reference implementation of the
+    `paper texture_nets_v2 <https://arxiv.org/abs/1701.02096>`_.
+  - ``texture_nets_v1``: Corresponds to the reference implementation of the
+    `paper texture_nets_v1 <https://arxiv.org/abs/1603.03417>`_.
+
+Unfortunately, the hyperparameters used differ from those in the papers, as well as
+among the individual implementations themselves. If you use ``instance_norm`` and
+``impl_params`` as in the following table, the appropriate parameters will be used:
+
+  - ``master``: The parameters specified in the implementation branch
+    `master <https://github.com/pmeier/texture_nets/tree/master>`_.
+  - ``texture_nets_v1``: The parameters specified in the implementation branch
+    `texture_nets_v1 <https://github.com/pmeier/texture_nets/tree/texture_nets_v1>`_.
+  - ``"paper"``: The parameters specified in the papers. Here ``instance_norm`` is only
+    used to use either :class:`~torch.nn.InstanceNorm2d` or
+    :class:`~torch.nn.BatchNorm2d`.
+
++-----------------+-----------+------------------------------------+
+|                 |           |        ``instance_norm``           |
++-----------------+-----------+-------------+----------------------+
+|                 |           |   ``True``  |       ``False``      |
++-----------------+-----------+-------------+----------------------+
+| ``impl_params`` | ``True``  | ``master``  | ``texture_nets_v1``  |
+|                 +-----------+-------------+----------------------+
+|                 | ``False`` | ``"paper"`` |      ``"paper"``     |
++-----------------+-----------+-------------+----------------------+
```

Comment:
I think this table is obsolete. It does not add any information to the explanation above.